dune-project

```diff
@@ -0,0 +1,2 @@
+(lang dune 1.0)
```

Comment:
We've got dune 1.10 now, so this could use a higher number here, but this is fine too.

Reply:
The idea with the (lang) version in dune is to have the minimum dune version required for the language features.  Dune will prompt you with an error message and the updated version if you use something newer, so this helps keep the version constraint as flexible as possible.